swahili/README.md

```diff
@@ -0,0 +1,28 @@
+# Kenya Swahili Braille Code
+
+## [Kenya Swahili Braille Code - Graded](Kenya%20Swahili%20Braille%20Code%20-%20Graded.docx)
+
+Document supplied by Martin Kieti, Chairman of Kenya Institute for the Blind.
+
+- _applies to_: Kenya
+- _language_: English
+
+Derived from the Kenya Institute of Curriculum Development's Swahili Braille Manual: “Mwongozo wa Breli Ya Kiswahili”;
```

Comment:
Do you know what exactly is the relationship between this document and the "Mwongozo wa Breli Ya Kiswahili" document?

Reply:
I understand this document is largely a translation into English of the Swahili document with a few ammendments resulting from the stakeholders' meeting. I have asked for a copy of the original Swahili document but to no avail so far. I think they someone will need to scan it.